packages/portal/spec-renderer/src/components/SpecOperationsList.vue

```diff
@@ -115,9 +115,13 @@
 <script setup lang="ts">
 import { PropType, computed, ref, watch, onMounted } from 'vue'
 import type { OperationListFilterFunction, Operation, OperationListItem, Tag } from '../types'
+import { v1 as uuidv1 } from 'uuid'
+import { createI18n } from '@kong-ui-public/i18n'
+import english from '../locales/en.json'
 import clonedeep from 'lodash.clonedeep'
-import OperationsListItem from './operations-list/OperationsListItem.vue'
+import useUtilities from '../composables/useUtilities'
```

Comment:
issue: for stubbing purposes, composables need to all be exported from the `/composables/index.ts` file like done here: https://github.com/Kong/shared-ui-components/blob/main/packages/konnect-app-shell/src/composables/index.ts

and then imported like `import composables from '../composables`